docs/setup-android.md

```diff
@@ -66,6 +66,8 @@ After installing the plugin and syncing your Android Studio project with Gradle,
 
 **For React Native >= v0.29**
 
+If you are using React Native for the whole application please do the following:
```

Comment:
Not sure if the word `whole application` fits good here. Probably we could rephrase like `If you are integrating Code Push into React Native application please do the following steps:`